public/New-DbaAvailabilityGroup.ps1

```diff
@@ -366,6 +381,14 @@ function New-DbaAvailabilityGroup {
             return
         }
 
+        if ($IsContained -and $server.VersionMajor -lt 16) {
+           Stop-Function -Level Warning -Message "Contained availability groups are only supported in SQL Server 2022 and above"
```

Comment:
Aa you can see some lines above, we need an additional "return" in case Stop-Function only issues a warning and not an exception (this depends on -EnableException). And please remove the `-Level Warning`. But you should add the `-Target $Primary` so that all the tests are written the same way.